assets/sass/patterns/atoms/definition-list.scss

```diff
@@ -1,5 +1,56 @@
 @import "../../definitions";
 
+.definition-list--timeline {
+  @include margin(24 0 0);
+  @include padding(24, "bottom");
+
+  @media only all and (min-width: #{get-rem-from-px($bkpt-site--wide)}em) {
+    border-bottom: 1px solid $color-text-dividers;
+  }
+
+  dt,
+  dd {
+    @include margin(8, "bottom");
+    font-family: $font-secondary;
+    font-weight: bold;
+
+    @media only all and (min-width: #{get-rem-from-px($bkpt-site--wide)}em) {
+      @include margin(0, "bottom");
+    }
+  }
+
+  dt {
+    @include font-size-and-vertical-height(14, 24);
+    @include margin(0, "top");
+
+    @media only all and (min-width: #{get-rem-from-px($bkpt-site--wide)}em) {
+      @include margin(0, "bottom");
+    }
+  }
+
+  dd {
+    @include font-size-and-vertical-height(12, 24);
+    color: $color-text-secondary;
+
+    a {
+      color: $color-text-secondary;
+    }
+
+    a::before {
+      content: "\00a0\2219\00a0";
+      font-weight: bold;
+    }
+
+    @media only all and (min-width: #{get-rem-from-px($bkpt-site--wide)}em) {
+      @include margin(16, "bottom");
```

Comment:
Is this margin correct as it's a strange number as we normally do them in multiples of 12 but can be halved so would have thought this would have been 18px. Can we check this with Chris please?

Reply:
I checked the design and you're right. I've updated the spaces.